src/main/java/Parser.java

```diff
@@ -0,0 +1,132 @@
+import java.util.ArrayList;
+import java.util.Arrays;
+
+// Deals with making sense of the user command
+public class Parser {
+
+    // Constructor
+    public Parser() { }
+
+    public Command parse(String userInput, TaskList taskList) throws DukeException {
```

Comment:
Maybe a comment here would be useful to explain what the parse method does generally, on top of the comments within the method